evm/src/proof.rs

```diff
@@ -101,14 +101,15 @@ pub struct BlockMetadata {
     pub block_number: U256,
     /// The difficulty (before PoS transition) of this block.
     pub block_difficulty: U256,
+    /// The `mix_hash` value of this block.
     pub block_random: H256,
-    /// The gas limit of this block. It must fit in a `u32`.
+    /// The gas limit of this block.
```

Comment:
```suggestion
    /// The gas limit of this block. It must fit in a `u64`.
```

Reply:
I hadn't mentioned it because other fields also implicitely have to fit in a u64, but will add a comment for all for clarity.